README.md

```diff
@@ -69,9 +69,16 @@ The **Format Tags** helper takes a comma separated list of tags and render them
 </ul>
 ```
 
-### Format time
+### Format time (uses moment.js)
 
-[ToDo]
+The **Format time** helper transform a date into a user-friendly format. It uses the [moment.js library](https://momentjs.com/docs/).
+
+The `date` and `format` parameters are documented respectively [here](https://momentjs.com/docs/#/parsing/string/) and [here](https://momentjs.com/docs/#/parsing/string-format/).
+
+**Example**
+```
+{{#formatTime date format}}
```

Comment:
you only need the `#` in handlebars if your helper has a body, for this, you can remove the `#`  
see http://handlebarsjs.com/block_helpers.html   
and the helper in use, within my blog: https://github.com/toh82/me/blob/master/src/partials/blogArticle.hbs